tools/ea/Ea/Genesis.hs

```diff
@@ -290,6 +308,15 @@ mainnet9 = mainnet0
     & txChainId .~ Nine
     & allocations .~ (Just mainnetAllocations9)
 
+mainnetKAD :: Genesis
+mainnetKAD = mainnet0
```

Comment:
Let's not base these off of 0 and probably just not use a default. In this case, it includes `mainnetKeysets` which we don't need, but it generally is hard for maintainability when it's so different from `mainnet0`.

Reply:
Addressed